binary_search_trees/array_to_bst.py

```diff
@@ -10,4 +10,11 @@ def arr_to_bst(arr):
         Balanced Binary Search Tree using the elements in the array.
         Return the root of the Binary Search Tree.
     """
-    pass
\ No newline at end of file
+    if len(arr) == 0:
+        return None
+
+    mid_node = len(arr)//2
```

Comment:
Tiny suggestion to rename this variable something like `mid_index` instead of `mid_node` since you are storing an index in the variable. This can help differentiate it from line 17 below where you are storing a Node object to readers of your code and yourself down the line!